framework/source/class/qx/ui/form/Spinner.js

```diff
@@ -118,6 +118,17 @@ qx.Class.define("qx.ui.form.Spinner",
     } else {
       this.initValue();
     }
+
+    // forward the focusin and focusout events to the textfield. The textfield
+    // is not focusable so the events need to be forwarded manually.
+    this.addListener("focusin", function(e) {
+      textField.fireNonBubblingEvent("focusin", qx.event.type.Focus);
+      textField.selectAllText();
```

Comment:
i think tab vs click differs because it's designed around larger text input areas - EG with a click, the implication is that you could be positioning the cursor at the same time as adding focus, and if you had a long text field or multi-lone text area that could be important; when using tab, the idea is that you are tabbing through a form filling in each field, so select-all is useful so that you can review and quickly replace the content.